doc/whats_new/_contributors.rst

```diff
@@ -176,4 +176,6 @@
 
 .. _Nicolas Hug: https://github.com/NicolasHug
 
-.. _Guillaume Lemaitre: https://github.com/glemaitre
\ No newline at end of file
+.. _Guillaume Lemaitre: https://github.com/glemaitre
+
+.. _Ying Xiong: https://github.com/yxiong
```

Comment:
This diff can be reverted.

This file is traditionally used for contributors that want to link to a non-github page or project maintainers. In `whats_new/v1.1.rst`, the `:user:Ying Xiong <yxiong>` directive will display your name and link to your github page.

Reply:
Done.